egs/librispeech/ASR/pruned_transducer_stateless2/scaling.py

```diff
@@ -52,7 +52,15 @@ def forward(
         if x.requires_grad:
             if channel_dim < 0:
                 channel_dim += x.ndim
-            sum_dims = [d for d in range(x.ndim) if d != channel_dim]
+
+            #  sum_dims = [d for d in range(x.ndim) if d != channel_dim]
+            # The above line is not torch scriptable for torch 1.6.0
+            # torch.jit.frontend.NotSupportedError: comprehension ifs not supported yet:  # noqa
+            sum_dims = 0
+            for d in range(x.ndim):
+                if d != channel_dim:
+                    sum_dims += d
```

Comment:
I think this is not equivalent to the original one, is it right?

Should be like this:
```
sum_dims = []
# If torch version less than 1.7.0, `if` in List will cause
# torch.jit.frontend.NotSupportedError: comprehension ifs not
# supported yet
if torch.jit.is_scripting() and torch.__version__ < '1.7.0':
    for d in range(x.ndim):
        if d != channel_dim:
            sum_dims.append(d)
else:
    sum_dims = [d for d in range(x.ndim) if d != channel_dim]
```

Correct me if I am wrong.

Reply:
thanks

Reply:
I verified  that this was fixed in torch 1.7.0. But it is ok to remove the if statement, just make it
``` 
sum_dims = []
for d in range(x.ndim):
        if d != channel_dim:
            sum_dims.append(d)
```